extension/services/applemusic/player.js

```diff
@@ -0,0 +1,49 @@
+/* globals helpers */
+
+this.player = (function() {
+    const SEARCH_PLAY = ".lookup__controls .play-button";
+  
+    class Player extends helpers.Runner {
+      action_play() {
+        const button = this.querySelector(SEARCH_PLAY);
+        button.click();
+      }
+  
+      async action_search({ query, thenPlay }) {
+        const searchInput = this.querySelector("#search-box .dt-search-box__input");
+        searchInput.focus();
+        this.setReactInputValue(searchInput, query);
+        const pressEnter = new KeyboardEvent("keydown", {
+            bubbles: true, cancelable: true, keyCode: 13
+        });
+       // document.dispatchEvent(pressEnter);
```

Comment:
Is the Enter necessary, or no? You might also be able to call `searchInput.form.submit()`

Reply:
Umm okay, I would try this instead.

Reply:
@ianb  the challenge is this search input is not embedded within a form.

Reply:
Whichever way you do it is fine if it works